wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -662,6 +689,12 @@ class KotlinGenerator private constructor(
     else -> baseClass.asNonNull()
   }
 
+  private fun MessageType.withOneOfs(): ImmutableList<Field> {
```

Comment:
How about `fieldsWithOneOfs()`? This method name is confusing.

I see where `ImmutableList` is coming from.. Can we wrap it into a `listOf()` to avoid referencing this type?